src/main/java/org/edumips64/utils/args/EduMipsVersion.java

```diff
@@ -0,0 +1,16 @@
+package org.edumips64.utils.args;
+
+import org.edumips64.utils.MetaInfo;
+import picocli.CommandLine;
+
+public class EduMipsVersion implements CommandLine.IVersionProvider {
+
+    @Override
+    public String[] getVersion() {
+        return new String[]{"EduMIPS64 version " + MetaInfo.VERSION +
```

Comment:
Would it be possible to move the code that generates this string to a static method?

The reason is that to get the text the new code creates a new `EduMipsVersion` instance just to call `getVersion()` and get this string, while the whole behavior doesn't really need the allocation of a new version, it could be something like  `EduMipsVersion.getVersionString();`.

Reply:
I will try that in the updates I am working on for this branch. Picocli requires using this style to allow dynamically generating the version, but I don't think it would break to add a static method. Will let you know if it doesn't work, otherwise will update.

Reply:
Sorry, I was not sufficiently clear. What I meant is that we can move the logic itself in a static method (or a `public static final string` really), and use it in `getVersion()`.

Reply:
Looks good, thanks!